docs/vars.md

```diff
@@ -189,11 +189,18 @@ before deploying dojot
 * *dojot_http_agent_port*: Port that will be used to communicate with HTTP Agent. Defaults to **8080**.
 * *dojot_kafka_port*: Port that will be used to communicate with Kafka. Defaults to **9092**.
 
-### - Metrics
+### - Metrics and Logging
 
 * *dojot_enable_locust_exporter*: Whether to activate the Locust Exporter or not. Defaults to **false**.
 * *dojot_locust_exporter.ip*: IP or hostname where there is a Locust Exporter running. Defaults to **127.0.0.1**.
 * *dojot_locust_exporter.port*: Port exposed by Locust Exporter. Defaults to **9646**.
+* *dojot_monitoring_namespace*: Namespace that will be created for monitoring stack deployment. Defaults to **dojot-monitoring**.
+* *dojot_helm_install*: Whether to install helm3 binary or not. Defaults to **true**.
```

Comment:
Is it correct to set the default value of this variable to true?

Reply:
Yes, because in general kubectl and helm are run from node master.

For example, if I'm going to use kubectl and helm from my machine I can disable this variable.

That was my ideia.